docs/pages/api-reference/operators.md

```diff
@@ -101,6 +101,19 @@ The `aggregate` operator has the following parameters:
 
 <pre snippet="api-reference/operators_ref#aggregate"></pre>
 
+### Groupby/ First
+
+Fennel allows you to groupby and get the first entry (by timestamp) in each group using the `first` operator.
+The api requires you to chain the `groupby` operator with the `first` operator.
```

Comment:
'API requires you to...' feels a bit weird from language POV. Instead, maybe simplify and say '`...first operator must always be preceeded by a groupby (else an error is raised)...`

Reply:
changed the wording. Let me know what you think now